app/about.html

```diff
@@ -44,52 +44,104 @@
 
   <main>
     <!-- About section -->
-    <section id="about">
+    <section id="about" class="inset-1220">
       <h1 class="uppercase section-title">About us</h1>
 
-      <div class="team-member">
-        <img src="./images/desktop.gif" alt="Sia Karamalegos">
-        <h2>Sia Karamalegos</h2>
-        <h3>Founder &amp; Lead Developer</h3>
-        <p>bio...</p>
-        <ul class="social">
-          <li>
-            <a href="#">github</a>
-          </li>
-          <li>
-            <a href="#">linkedin</a>
-          </li>
-          <li>
-            <a href="#">medium</a>
-          </li>
-          <li>
-            <a href="#">twitter</a>
-          </li>
-        </ul>
+      <div class="bio">
+        <div class="bio-image">
+          <img src="images/sia.png" alt="Sia head shot">
+        </div>
+        <div class="bio-description">
+          <h2>Sia Karamalegos</h2>
+          <h3>Founder &amp; Lead Developer</h3>
+          <p>Sia has over 15 years of experience in technology, strategy, and operations across multiple industries. She loves
+            building applications that help make people's lives easier. She also loves making developer's lives easier through
+            teaching, speaking at conferences, and mentoring.
+          </p>
+          <p>In both 2015 and 2016, Sia was recognized in the Silicon Bayou 100. She graduated from Texas A&amp;M University
+            with a BS in chemical engineering and received her MBA from Harvard Business School.
+          </p>
+          <p>When she's not coding, speaking, or consulting, Sia likes to design crochet patterns and dabble in charcoal figure
+            drawing. She's also an avid endurance athlete.</p>
+          <ul class="bio-social">
+            <li>
+              <a href="https://github.com/siakaramalegos" target="_blank">
+                <i class="fa fa-github" aria-hidden="true"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/karamalegos" target="_blank">
+                <i class="fa fa-linkedin-square" aria-hidden="true"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://medium.com/@thegreengreek" target="_blank">
+                <i class="fa fa-medium" aria-hidden="true"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://twitter.com/thegreengreek/" target="_blank">
```

Comment:
Small detail: I'd recommend reversing the order of Medium and Twitter here. Since you have Medium and I have Dribbble, having our sets of common icons follow the same order makes sense to me (GitHub, LinkedIn, Twitter, then other).

Reply:
yeah i should have noticed that. originally we both had twitter last, but i changed yours and forgot mine. fixed.